buildcatrust/certstore_output.py

```diff
@@ -1,4 +1,5 @@
 # SPDX-FileCopyrightText: 2021 Luke Granger-Brown <git@lukegb.com>
+# 2024 Ryan Lahfa <ryan@lahfa.xyz>
```

Comment:
This isn't compatible with SPDX; replace the copyright attribution with `2021 the buildcatrust authors` and create an AUTHORS file in the root.

Reply:
https://community.kde.org/Guidelines_and_HOWTOs/Licensing seemed to indicate this was fine but I can do that no problem

Reply:
I implemented your suggestion.